python/python/glide/async_commands/cluster_commands.py

```diff
@@ -514,3 +516,52 @@ async def flushall(
             TClusterResponse[TOK],
             await self._execute_command(RequestType.FlushAll, args, route),
         )
+
+    async def cluster_scan(
```

Comment:
If this function represents a redis command (it is), so it should be named `scan`
```suggestion
    async def scan(
```

Reply:
it need to be discussed, since it is not exactly scan and im not sure we dont want to give the user the option to send normal scan. It is a feature of the library that's behave different then the scan command. it is not exactly a "redis command".